README.md

```diff
@@ -24,9 +24,17 @@ What is working :
 - trackpad
 - display brightness/ brightness control (F11 & F12)
 - battery managment
+- HDMI (Video and sound)
 
 Note : Sound on headphones is also working, please follow 'Headphones sound fix' down below. 
 
+Note 2: To HDMI work perfectly, it is necessary:
```

Comment:
This note is too long ! Please keep it short and straight.

Reply:
Ok.